framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/Modeling/AbpEntityTypeBuilderExtensions.cs

```diff
@@ -302,6 +304,25 @@ public static void ConfigureFullAuditedAggregateRoot<T>(this EntityTypeBuilder<T
             b.As<EntityTypeBuilder>().TryConfigureConcurrencyStamp();
         }
 
+        public static void ConfigureMultiLingual<T, TTranslation>(this EntityTypeBuilder<T> b)
+            where T : class, IHasMultiLingual<TTranslation>
+            where TTranslation : class, IMultiLingualTranslation
+        {
+            b.As<EntityTypeBuilder>().TryConfigureEntityTranslation();
+        }
+
+        public static void TryConfigureEntityTranslation(this EntityTypeBuilder b)
+        {
+            if (b.Metadata.ClrType.IsAssignableTo<IMultiLingualTranslation>())
+            {
+                b.HasIndex(nameof(IMultiLingualTranslation.Language))
+                    .IsUnique();
```

Comment:
What about multi-tenancy? Making this unique prevents multiple tenants to insert entity with the same language name.

Reply:
Combination of Tenant & Language can be unique @realLiangshiwei